packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -348,6 +348,7 @@ cp -pa .%{gem_dir}/* \
 %{foreman_assets_plugin}
 %{foreman_webpack_foreman}
 %{foreman_webpack_plugin}
+%exclude %{foreman_webpack_plugin}/*.js.map
```

Comment:
ACK for this. Is Foreman not also running into this same issue?

Reply:
That surprised me. My theory is that it doesn't mix foreman and katello locations so gets away with it by using relative paths.

It also looks like this isn't enough, probably because the check runs before selecting files.